ACHP/VICompressorTello.py

```diff
@@ -0,0 +1,271 @@
+from __future__ import division, print_function, absolute_import
```

Comment:
@davideziviani 
Do you want to add VI comp cycle!?

Reply:
@abahman nope, not now. But this model is available in the public domain already.